docker/compose/faucet-validator-fullnode/keys/generate_keys_and_config.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
```

Comment:
if you're interested in this path, I would recommend we push this into scripts, but the historical intent was that this should be handled invisibly from the user and be platform independent -- which currently our docker compose strategy is. The current method to do this is [here](https://github.com/diem/diem/blob/master/diem-node/src/lib.rs#L114). Another reason is reduction ins testable surfaces. As you well are aware, much of this is not in our CI/CD and is extremely fragile. Hopefully we don't see anything new happening in this space, but I actually could imagine a case where we invest time in improving our genesis framework and then this might not work any more.